lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
```

Comment:
I'd love to move to requiring user-agent in our clients. That could be a separate PR, tho.